### ISSUE TITLE
New problems for settings import

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#3722]
Currently, `kalite.distributed.settings` raise an ImportError when `kalite.settings` tries to import it.

```
>>> from kalite import settings
kalite.distributed
ImportError <type 'exceptions.ImportError'>
Traceback (most recent call last):
  File ".virtualenvs/kalite_pip/share/kalite/python-packages/fle_utils/settingshelper.py", line 65, in import_installed_app_settings
    app_settings = __import__(app + ".settings")
  File ".virtualenvs/kalite_pip/local/lib/python2.7/site-packages/kalite/distributed/settings.py", line 164, in <module>
    version_info = version.VERSION_INFO()
  File ".virtualenvs/kalite_pip/local/lib/python2.7/site-packages/kalite/version.py", line 29, in VERSION_INFO
    return open_json_or_yml(os.path.join(settings.CONTENT_DATA_PATH, "version.yml"))
  File ".virtualenvs/kalite_pip/local/lib/python2.7/site-packages/kalite/shared/utils.py", line 24, in open_json_or_yml
    import yaml
ImportError: No module named yaml
```

This is because we want to use version data in our settings:

https://github.com/learningequality/ka-lite/blob/develop/kalite/distributed/settings.py#L162

It would be great to burn this down, since we don't actually need to be using a git commit hash for cache key prefixes.

**Comment by benjaoming** [comment]
First part fixed in bcf2863

**Comment by MCGallaspy** [comment]
How do I reproduce this? `from kalite import settings` works without error for me, using the `shell_plus` command.

**Comment by benjaoming** [comment]
@MCGallaspy the fix was already merged into develop at the time whe you tested, perhaps that's the problem. No matter what, you'd need to be running without pyyaml.

I'll have a second look at develop to make sure that there's no overlapping or brainfarts from my side.

**Comment by aronasorman** [comment]
Looks like it's fixed for the current 0.14.x.
